src/packet.rs

```diff
@@ -132,20 +177,65 @@ impl TryFrom<usize> for ContentFormat {
+            51 => Ok(ContentFormat::ApplicationJsonPathJson),
+            52 => Ok(ContentFormat::ApplicationMergePathJson),
             60 => Ok(ContentFormat::ApplicationCBOR),
+            61 => Ok(ContentFormat::ApplicationCWt),
+            62 => Ok(ContentFormat::ApplicationMultipartCore),
+            63 => Ok(ContentFormat::ApplicationCborSeq),
+            96 => Ok(ContentFormat::ApplicationCoseCoseTypeCoseEncrpty),
+            97 => Ok(ContentFormat::ApplicationCoseCoseTypeCoseMac),
+            98 => Ok(ContentFormat::ApplicationCoseCoseTypeCoseSign),
+            101 => Ok(ContentFormat::ApplicationCoseKey),
+            102 => Ok(ContentFormat::ApplicationCoseKeySet),
             110 => Ok(ContentFormat::ApplicationSenmlJSON),
             111 => Ok(ContentFormat::ApplicationSensmlJSON),
             112 => Ok(ContentFormat::ApplicationSenmlCBOR),
             113 => Ok(ContentFormat::ApplicationSensmlCBOR),
             114 => Ok(ContentFormat::ApplicationSenmlExi),
             115 => Ok(ContentFormat::ApplicationSensmlExi),
+            140 => Ok(ContentFormat::ApplicationYangDataCborIdSid),
+            256 => Ok(ContentFormat::ApplicationCoapGroupJson),
+            271 => Ok(ContentFormat::ApplicationDotsCbor),
+            272 => Ok(ContentFormat::ApplicationMissingBlocksCborSeq),
+            280 => Ok(ContentFormat::ApplicationPkcs7MimeSmimeTypeServerGeneratedKey),
+            281 => Ok(ContentFormat::ApplicationPkcs7MimeSmimeTypeCertsOnly),
+            284 => Ok(ContentFormat::ApplicationPkcs8),
+            285 => Ok(ContentFormat::ApplicationCsrattrs),
+            286 => Ok(ContentFormat::ApplicationPkcs10),
+            287 => Ok(ContentFormat::ApplicationPkixCert),
+            290 => Ok(ContentFormat::ApplicationAifCbor),
+            291 => Ok(ContentFormat::ApplicationAifJson),
             310 => Ok(ContentFormat::ApplicationSenmlXML),
             311 => Ok(ContentFormat::ApplicationSensmlXML),
+            320 => Ok(ContentFormat::ApplicationSenmlEtchJson),
+            322 => Ok(ContentFormat::ApplicationSenmlEtchCbor),
+            340 => Ok(ContentFormat::ApplicationYangDataCbor),
+            341 => Ok(ContentFormat::ApplicationYangDataIdName),
+            432 => Ok(ContentFormat::ApplicationTdJson),
+            836 => Ok(ContentFormat::ApplicationVoucherCoseCbor),
+            10000 => Ok(ContentFormat::ApplicationVndOcfCbor),
+            10001 => Ok(ContentFormat::ApplicationOscore),
+            10002 => Ok(ContentFormat::ApplicationJavascript),
+            11050 => Ok(ContentFormat::ApplicationJsonDeflate),
+            11060 => Ok(ContentFormat::ApplicationCborDeflate),
+            11542 => Ok(ContentFormat::ApplicationVndOmaLwm2mTlv),
+            11543 => Ok(ContentFormat::ApplicationVndOmaLwm2mJson),
+            11544 => Ok(ContentFormat::ApplicationVndOmaLwm2mCbor),
+            20000 => Ok(ContentFormat::ApplicationTextCss),
+            30000 => Ok(ContentFormat::ApplicationSvgXml),
```

Comment:
These aren't `application/*` according to [the registry](https://www.iana.org/assignments/core-parameters/core-parameters.xhtml#content-formats), right? Rather `text/css` and `image/svg+xml`.